go-controller/pkg/ovn/address_set.go

```diff
@@ -259,3 +393,19 @@ func (as *ovnAddressSet) Destroy() error {
 	as.ips = nil
 	return nil
 }
+
+func getIPv4ASNameForNamespace(namespaceName string) string {
+	return namespaceName + ipv4AddressSetSuffix
```

Comment:
for all 4 of these functions, neither the function name nor the arg should refer to "namespace"

Reply:
Done